NanoProd/crab/Run3_2022_postEE/data.yaml

```diff
@@ -2,19 +2,24 @@ config:
   params:
     sampleType: data
     era: Run3_2022_postEE
```

Comment:
```suggestion
    era: Run3_2022FG
```

Reply:
Should this really be different from your suggestion above for `Run3_2022EE` in the MC configs?

Reply:
As an alternative, we can change it on the [RunKit side](https://github.com/kandrosov/RunKit/blob/main/nanoProdWrapper.py#L63-L64). What would you prefer?

Reply:
Now I remember why I made it different: because based on [PdmV summary table](https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun3Analysis#2022_Analysis_Summary_Table) MC and data splitting are not aligned: CDE eras have the same GT for data.